build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -267,9 +267,9 @@ public static BuildTargetResult GenerateMSbuildPropsFile(BuildTargetContext c)
             msbuildProps.AppendLine($"    <HostVersion>{hostVersion.LatestHostPolicyVersion.WithoutSuffix}</HostVersion>");
             msbuildProps.AppendLine($"    <HostResolverVersion>{hostVersion.LatestHostFxrVersion.WithoutSuffix}</HostResolverVersion>");
             msbuildProps.AppendLine($"    <HostPolicyVersion>{hostVersion.LatestHostVersion.WithoutSuffix}</HostPolicyVersion>");
-            msbuildProps.AppendLine($"    <BuildNumberMajor></BuildNumberMajor>");
-            msbuildProps.AppendLine($"    <BuildNumberMinor></BuildNumberMinor>");
-            msbuildProps.AppendLine($"    <PreReleaseLabel></PreReleaseLabel>");
+            msbuildProps.AppendLine($"    <BuildNumberMajor>{hostVersion.LatestHostBuildMajor}</BuildNumberMajor>");
+            msbuildProps.AppendLine($"    <BuildNumberMinor>{hostVersion.LatestHostBuildMinor}</BuildNumberMinor>");
```

Comment:
@schellap - can you answer that?

FYI - this is what is currently in 'master':  https://github.com/dotnet/core-setup/blob/master/build_projects/dotnet-host-build/CompileTargets.cs#L263-L274


This diff was me fixing up the bad merge from the previous merge commit.